lwallet/Command.hs

```diff
@@ -111,6 +113,20 @@ switch = lexeme $ positive $> True <|>
     positive = text "+" <|> text "y" <|> text "yes"
     negative = text "-" <|> text "n" <|> text "no"
 
+stakeholderId :: Parser StakeholderId
+stakeholderId = lexeme $ do
+    str <- many1 alphaNum
+    case decodeTextAddress (toText str) of
```

Comment:
There's a bigger problem then. `TxOutDistribution` is a `[(StakeholderId, Coin)]`. How can the lwallet user get `StakeholderId`s? Currently the only thing they have is `listaddr` which lists addresses and PKs.